src/Mod/Sketcher/Gui/TaskSketcherGeneral.ui

```diff
@@ -14,6 +14,8 @@
    <string>Form</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <layout class="QHBoxLayout" name="horizontalLayout">
```

Comment:
Please fix indentation in the file.

Reply:
Preferences/Sketcher/General is correct. Grid size changeable even if grid disabled there.
I fixed the indentation in last commit

Reply:
Edit: Instead I squashed the 2 commits and force pushed the new commit.